newIDE/app/src/InstancesEditor/index.js

```diff
@@ -47,6 +48,7 @@ export type InstancesEditorPropsWithoutSizeAndScroll = {|
   onInstancesMoved: (instances: Array<gdInitialInstance>) => void,
   onInstancesResized: (instances: Array<gdInitialInstance>) => void,
   onInstancesRotated: (instances: Array<gdInitialInstance>) => void,
+  onAdditionalServiceComplete: (infoBarDetails: InfoBarDetails) => void,
```

Comment:
Same as `InstancesAdder`, let's not make InstancesEditor dependent on SceneEditor, and let's not make it less reusable by coupling it with a complex "InfoBarDetails" and what not stuff.
See how it's done for existing "events": `onInstancesMoved`, `onInstancesSelected`... there is even something called `onInstancesAdded`!

You probably want to use this :) Surely you'll have to pass the array of added instances as an argument (which is fair).